main.js

```diff
@@ -3386,7 +3396,10 @@ function startInstance(id, wakeUp) {
                                     logger.debug(`${hostLogPrefix} Initialize crash timer of ${id}`);
                                     procs[id].crashResetTimer = setTimeout(() => {
                                         logger.debug(`${hostLogPrefix} Cleared crash counter of ${id}, because 10 minutes no crash`);
-                                        procs[id].crashCount = 0;
```

Comment:
We should also set procs[id].crashResetTimer = null when it is fired

Reply:
we always clear in exit handler, on crash https://github.com/ioBroker/ioBroker.js-controller/pull/1129/files#diff-58417e0f781b6656949d37258c8b9052ed266e2eb7a5163cad7b0863e6b2916aR3384-R3389 here and immediately set timer again because crash ;-) so no need to delete

on non-crash exit it will also be cleared and w/o crash no timer will be started again https://github.com/ioBroker/ioBroker.js-controller/pull/1129/files#diff-58417e0f781b6656949d37258c8b9052ed266e2eb7a5163cad7b0863e6b2916aR3400-R3403